security-command-center/snippets/src/test/java/vtwo/FindingsIT.java

```diff
@@ -114,20 +120,32 @@ public void testListAllFindings() throws IOException {
   public void testListFilteredFindings() throws IOException {
     ListFindingsWithFilter.listFilteredFindings(ORGANIZATION_ID, SOURCE.getName().split("/")[3],
         LOCATION);
+
     assertThat(stdOut.toString()).contains(FINDING_1.getName());
     assertThat(stdOut.toString()).doesNotContain(FINDING_2.getName());
   }
 
   @Test
   public void testGroupAllFindings() throws IOException {
     GroupFindings.groupFindings(ORGANIZATION_ID, SOURCE.getName().split("/")[3], LOCATION);
+
     assertThat(stdOut.toString()).contains("Listed grouped findings.");
   }
 
   @Test
   public void testGroupFilteredFindings() throws IOException {
     GroupFindingsWithFilter.groupFilteredFindings(ORGANIZATION_ID, SOURCE.getName().split("/")[3],
         LOCATION);
+
     assertThat(stdOut.toString()).contains("count: 1");
   }
+
+  @Test
+  public void testSetFindingsByStateInactive() throws IOException {
+    SetFindingsByState.setFindingState(ORGANIZATION_ID, LOCATION, SOURCE.getName().split("/")[3],
+        FINDING_1.getName().split("/")[7]);
+
+    assertThat(stdOut.toString()).contains("Updated Finding: ");
```

Comment:
Change this to assert the state in the returned response from `SetFindingsByState.setFindingState`.

Reply:
done